tests/moe/test_w4a8_swiglu.py

```
@@ -0,0 +1,121 @@
+_CLAMPS = [None, (1.702, 1.0, 0.5)]
+
+
+@pytest.mark.skipif(
+    not _OK
+    or not torch.cuda.is_available()
+    or not is_sm90a_supported(torch.device("cuda")),
+    reason="needs SM90a + cute_dsl",
+)
+@pytest.mark.parametrize("M,K,I", [(128, 256, 256), (256, 512, 512), (512, 1024, 1024)])
+@pytest.mark.parametrize("clamp", _CLAMPS)
+def test_w4a8_swiglu(M, K, I, clamp):
```

Ruff (0.15.15)

[error] Ambiguous variable name: `I`

(E741)